programming-extensions/programming-extension-vfsprovider/src/main/java/org/objectweb/proactive/extensions/vfsprovider/client/ProActiveFileObject.java

```diff
@@ -850,6 +859,51 @@ private static int getConfiguredNumberOfThreads() {
         }
     }
 
+    public static final class ProActiveVfsComponentContext implements VfsComponentContext {
+
+        private FileSystemManager manager;
+
+        public ProActiveVfsComponentContext(FileSystemManager manager) {
+            this.manager = manager;
+        }
+
+        @Override
+        public FileObject resolveFile(FileObject baseFile, String name, FileSystemOptions fileSystemOptions)
+                throws FileSystemException {
+            return null;
+        }
+
+        @Override
+        public FileObject resolveFile(String name, FileSystemOptions fileSystemOptions) throws FileSystemException {
+            return null;
+        }
```

Comment:
Isn't that better to overload the method resolveFile by adding the missing parameter last in the list of parameters.
i.e
` public FileObject resolveFile(String name, FileSystemOptions fileSystemOptions)`
` public FileObject resolveFile(String name, FileSystemOptions fileSystemOptions, FileObject baseFile)`

Reply:
Well I guess those methods come from VfsComponentContext...

Reply:
yes, I have no freedom in the method definition